src/encoded/schemas/reference.json

```diff
@@ -287,6 +287,14 @@
             "title": "Project",
             "type": "string"
         },
+        "elements_selection_method": {
+            "title": "Elements selection method",
+            "type": "array"
+        },
+        "examined_loci.gene.symbol": {
```

Comment:
I think this is just `examined_loci.symbol`, and it would be of type “string” (`examined_loci` is an array, but `examined_loci[x].symbol` is a string). Actually, I’m not sure why this `columns` has types when most (but not all) other schema’s `columns` properties have no type.

Reply:
ah thanks. yep, I think it should be examined_loci.symbol instead of examined_loci.gene.symbol.